### PR TITLE
DSERV-192 Merging regulatory regions and ccre collections

### PR DESCRIPTION
**Pull request by pedrohr**

### CODE REVIEW

File: data/adapters/encode_enhancer_gene_adapter.py [hunk 1]
```diff
@@ -105,9 +105,17 @@ def process_file(self):
                         'chr': chr,
                         'start': start,
                         'end': end,
-                        'class': class_name,
+                        'type': 'candidate_cis_regulatory_element',
                         'source': self.source,
                         'source_url': self.source_url
                     }
 
+                    if class_name != 'enhancer':
```
Comment (mingjiecn) [inline data/adapters/encode_enhancer_gene_adapter.py:113]: should be `if class_name == 'enhancer':`

**Review by mingjiecn — CHANGES_REQUESTED** [review]
Also accessible_via and properties need to be updated in regulatory region schema.

File: data/adapters/encode_enhancer_gene_adapter.py (hunk 1 above)
Comment (pedrohr) [reply data/adapters/encode_enhancer_gene_adapter.py:113]: oh good catch!

**Review by mingjiecn — APPROVED** [review]

**Comment by mingjiecn** [comment]
One more thing. Can you update the collection key definition here since we merge two schemas? https://github.com/IGVF-DACC/igvf-catalog/blob/main/docs/collection_key_definition.md
Thanks!

**Pull request merged**